From 8d346e4f495938ac50bf38a75b53b49ea05b5b98 Mon Sep 17 00:00:00 2001
From: Raito Bezarius <masterancpp@gmail.com>
Date: Sat, 13 Sep 2025 15:31:05 +0200
Subject: [PATCH] =?UTF-8?q?lixPackageSets.lix=5F2=5F93.lix:=20restore=20lo?=
 =?UTF-8?q?wdown=20=E2=89=A5=201.4=20patch?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This was removed accidentally in
5d4210ba4e9e0a5b100c3b84c7e5c8b6e17f5aa0 ("lixPackageSets.lix_2_92:
drop").

Thanks to @magneticflux- for flagging this.

Change-Id: I731fafbe1cb47202110f359536c3023652b61dce
Signed-off-by: Raito Bezarius <masterancpp@gmail.com>
---
 pkgs/tools/package-management/lix/default.nix | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/pkgs/tools/package-management/lix/default.nix b/pkgs/tools/package-management/lix/default.nix
index 727ae28b0feb3f3a980498ff2fb2e4ee2ac433a8..4573325396ddfbdf5188ad604b12fed6fbdbf111 100644
--- a/pkgs/tools/package-management/lix/default.nix
+++ b/pkgs/tools/package-management/lix/default.nix
@@ -159,6 +159,15 @@ lib.makeExtensible (
           inherit src;
           hash = "sha256-YMyNOXdlx0I30SkcmdW/6DU0BYc3ZOa2FMJSKMkr7I8=";
         };
+
+        patches = [
+          # Support for lowdown >= 1.4, https://gerrit.lix.systems/c/lix/+/3731
+          (fetchpatch2 {
+            name = "lix-lowdown-1.4.0.patch";
+            url = "https://git.lix.systems/lix-project/lix/commit/858de5f47a1bfd33835ec97794ece339a88490f1.patch";
+            hash = "sha256-FfLO2dFSWV1qwcupIg8dYEhCHir2XX6/Hs89eLwd+SY=";
+          })
+        ];
       };
     };
 
